src/routes/volunteers.js

```diff
@@ -0,0 +1,11 @@
+const router = require('express').Router();
+// const { isAuth } = require('../middlewares/auth');
+// const { request } = require('chai');
+
+const volunteersController = require('../controllers/volunteers');
+
+router.get('/', volunteersController.getVolunteers);
+
+router.get('/:id', volunteersController.getVolunteers);
```

Comment:
great work, but I believe the second controller name should be `getVolunteer` not `getVolunteers`